index.html

```diff
@@ -320,16 +344,39 @@ <h3> Normative Specifications </h3>
               </p>
 
             </dd>
+            <dt> <a href="https://webscreens.github.io/window-placement/">Multi-Screen Window Placement</a></dt>
+            <dd>
+              <p>
+                An API that allows a web application to query its device for
+                information about directly connected displays, and additional APIs
+                to position windows relative to those displays.  Networked
+                presentation displays are out of scope for this API.</p>
+              <p class="draft-status"> <b>Draft state:</b> Editor's Draft</p>
+              <p>
+                This specification was incubated in the Second Screen Community
+                Group from Oct 2019 to Sep 2021, and was adopted by the Second
+                Screen Working Group as a deliverable in Sep 2021.  The Working
+                Group will publish the specification as an Editor's Draft after
+                rechartering is complete.
+              </p>
+              <p class="milestone">
+                <b>Expected completion:</b> The Working Group plans to publish
+                the Multi-Screen Window Placement as a final Recommendation
+                when its Success Criteria are met.  The timeline for this
+                charter proposes that the specification reach Candidate
+                Recommendation by Q4 2023.
```

Comment:
I think it is OK to be very brief here with a guesstimate of completion. The WG or the editor will not be punished for not meeting that milestone :-)

```suggestion
                <b>Expected completion:</b> Candidate Recommendation by Q4 2023
```